src/3D/Implementations/TempleInterior.cpp

```diff
@@ -136,7 +138,10 @@ void TempleInterior::Activate()
 
 	Locator::rendereringSystem::emplace<ecs::systems::RenderingSystemTemple>();
 	camera.SetOrigin(_templePosition);
-	camera.SetFocus(_templePosition + glm::quat(_templeRotation) * glm::vec3(0.0f, 0.0f, 1.0f));
+	const auto aspect = Locator::windowing::has_value() ? Locator::windowing::value().GetAspectRatio() : 1.0f;
+	camera.SetProjectionMatrixPerspective(100.f, aspect, config.cameraNearClip, config.cameraFarClip);
+	Locator::cameraPathSystem::value().Start(entt::hashed_string("temple/main"));
```

Comment:
@bwrsandman Default fov is definitely wrong but setting the fov to 100 almost matches vanilla. There's more to it than just fov as you'll see.

Reply:
Do you think it's wrong in the overworld too?

Reply:
You've run into a bug that is corrected with https://github.com/openblack/openblack/pull/788.
The angle should be 90 and with that PR, the image should match.